quinn-proto/src/connection/mod.rs

```diff
@@ -1230,7 +1230,9 @@ impl Connection {
         self.spaces[self.highest_space].ping_pending = true;
     }
 
-    #[doc(hidden)]
+    /// Update traffic keys
+    ///
+    /// The reason this is public is primarily for testing purposes.
```

Comment:
I feel like we should ditch this note? It seems like this could be useful in other kinds of scenarios too, and I don't think this note is helpful to our users.

Reply:
What other situations?

I think API docs should generally provide context describing, or some sort of breadcrumb for, when and how they should be used. This is pretty mysterious without that, and in crypto that can be dangerous.

Reply:
That's fair, but the "The reason this is public" wording is a little roundabout for that kind of thing? Maybe something like "This can be used to help test key updates, which normally happen after a long time."?

Reply:
Changed to

```
/// This can be useful for testing key updates, as they otherwise only happen infrequently.
```

Reply:
Changed to:

```rs
    /// Update traffic keys spontaneously
    ///
    /// This can be useful for testing key updates, as they otherwise only happen infrequently.
```